docs/configuring-playbook-synapse.md

```diff
@@ -17,6 +17,51 @@ Alternatively, **if there is no pre-defined variable** for a Synapse setting you
 
 - or, if extending the configuration is still not powerful enough for your needs, you can **override the configuration completely** using `matrix_synapse_configuration` (or `matrix_synapse_configuration_yaml`). You can find information about this in [`roles/matrix-synapse/defaults/main.yml`](../roles/matrix-synapse/defaults/main.yml).
 
+## Synapse OpenID Connect
+
+If you plan to rely only on SSO and don't want your users to ever login with password credentials disable this:
+```yaml
+matrix_synapse_password_config_enabled: true
+```
+
+Add this to allow seamless forwarding to element web app and element android app. Without this setting matrix will ask the user if he trusts the app he tries to login.  
+
+```yaml
+matrix_synapse_sso_client_whitelist:
+  - "https://element.{{ matrix_domain }}/"
+  - element://element
+
+```
+
+If you use `nginx proxy role` then you should add this setting as well to successfully pass redirects: 
+```yaml
+matrix_nginx_proxy_proxy_matrix_client_api_forwarded_location_synapse_oidc_api_enabled: true
```

Comment:
Can we also automatically toggle this via `group_vars/matrix_servers` based on some other variable?

```yaml
matrix_nginx_proxy_proxy_matrix_client_api_forwarded_location_synapse_oidc_api_enabled: "{{ matrix_synapse_oidc_enabled if matrix_synapse_enabled else false }}"
```

It makes things easier for people.